src/hn/ui/CommentItem.ui

```diff
@@ -58,34 +57,24 @@
           <class name="comment-item-body"/>
         </style>
       </object>
-      <packing>
-        <property name="expand">False</property>
-        <property name="fill">True</property>
-        <property name="position">0</property>
-      </packing>
     </child>
     <child>
       <object class="GtkRevealer" id="replies_container">
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
+        
+        <property name="can_focus">0</property>
```

Comment:
Generally you want to remove places where you had `can_focus` set to `False` on containers. Otherwise you won't be able to focus on anything within these containers.